### ISSUE TITLE
Filepicker Widget in Mux?

### ISSUE THREAD

**Issue by versipellis** [JuliaGizmos/Interact.jl#316]
I'm trying to use `filepicker` with Mux, and haven't been able to get it to work. I just noticed this line in the Tutorial, however:
```
Important note: this app needs to run in Blink, as the browser doesn't allow us to get access to the local path of a file.
```

Does this mean that the `filepicker` widget only works with Blink, or am I misunderstanding what the limitations are?

**Comment by piever** [comment]
That is correct, browsers don't have access to the local file system, so the "Blink" filepicker that returns the path to the selected file is not possible (normally this is handled server side). I'm not sure what's the best way to handle this in the Interact framework.

**Comment by versipellis** [comment]
Is there a way to load and cache that file into the server rather than a reference to the local file system?
